src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,53 @@
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__item {
+  margin-left: 20px;
+
+  &:first-child {
+    margin-left: 0;
+  }
+}
+
+.nav__link {
+  position: relative;
+
+  display: flex;
+  align-items: center;
+
+  height: 60px;
```

Comment:
The height of an element should not be fixed unless it is absolutely necessary. Fixed heights can lead to overflow issues with content and are generally not responsive. Consider using padding or min-height for a more flexible layout.